src/main/java/commands/AddEventCommand.java

```diff
@@ -0,0 +1,83 @@
+package commands;
+
+import skibidi.Command;
+import skibidi.SkibidiException;
+import skibidi.Ui;
+import storage.TaskStorage;
+import storage.Event;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a command to add an event task.
+ */
+public class AddEventCommand extends Command {
+    private final String description;
+    private final String startTime;
+    private final String endTime;
+
+    /**
+     * Creates a new AddEventCommand.
+     *
+     * @param input The input string containing the event description, start time and end time.
+     * @throws SkibidiException If the input string is in an invalid format.
+     */
+    public AddEventCommand(String input) throws SkibidiException{
+        String[] partsFrom = input.split("/from ");
+
+        if (partsFrom.length < 2) {
+            throw new SkibidiException("Invalid event format. " +
+                    "Usage: event [description] /from [start time] /to [end time]");
+        }
+
+        String[] partsTo = partsFrom[1].split("/to ");
+        if (partsTo.length < 2) {
+            throw new SkibidiException("Invalid event format. " +
+                    "Usage: event [description] /from [start time] /to [end time]");
+        }
+
+        this.description = partsFrom[0].substring(6).trim();
+        this.startTime = parseDateTime(partsTo[0].trim());
+        this.endTime = parseDateTime(partsTo[1].trim());
+
```

Comment:
There are a few blocks of empty lines here and there, just a small issue but worth taking note!